javascript/javascript2/week1/danishletter.js

```diff
@@ -0,0 +1,32 @@
+
+function findDanishLetter(danishString) {
+    const Danishletter = { count: 0, å: 0, ø: 0, æ: 0 };
+    const wordArray = danishString.split("");
+    const result = {};
+    for (item of wordArray) {
+        if (item === 'å') {
+            Danishletter.count += 1;
+            Danishletter.å += 1;
+        }
+        else if (item === 'ø') {
+            Danishletter.count += 1;
+            Danishletter.ø += 1;
+        }
+        else if (item === 'æ') {
+            Danishletter.count += 1;
+            Danishletter.æ += 1;
+        }
+    }
```

Comment:
Its ok to use if else if. suppose if you have like 10 letters in danish words you are going to write if..else so many times. So try to use what you learn recently like array, array methods and try to use them during exercises.